lucene/misc/src/java/org/apache/lucene/store/NativeUnixDirectory.java

```diff
@@ -47,10 +47,10 @@
  *
  * <p>To use this you must compile
  * NativePosixUtil.cpp (exposes Linux-specific APIs through
- * JNI) for your platform, by running <code>ant
- * build-native-unix</code>, and then putting the resulting
- * <code>libNativePosixUtil.so</code> (from
- * <code>lucene/build/native</code>) onto your dynamic
+ * JNI) for your platform, by running <code>
+ * ./gradlew build -Pbuild.native=true</code>, and then putting the resulting
```

Comment:
I removed the flag and forgot to update the docs here.

Reply:
Updated.